aptos-move/framework/aptos-names/sources/config.move

```diff
@@ -60,6 +63,11 @@ module aptos_names::config {
         set_tokendata_description(framework, string::utf8(b"This is an official Aptos Foundation Name Service Name"));
         set_tokendata_url_prefix(framework, string::utf8(b"https://aptosnames.com/api/v1/metadata/"));
 
+        // TODO: SET THIS TO SOMETHING REAL
```

Comment:
cc @CapCap. Can we consider finishing all of these TODOs? This makes the code seems unready for production